docs/docs/feature-library/teams-view-enhancements.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_label: teams-view-enhancements
+title: teams-view-enhancements
+---
+
+The Teams View can be modified by adding and removing columns in the [WorkersDataTable](https://www.twilio.com/docs/flex/developer/ui/components#add-columns-to-workersdatatable). 
```

Comment:
there are a set of standard sections we want to add to the docs page.  The main one missing here is "flex user experience" typically its a gif but in this case i think a screenshot would do